packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -69,6 +68,36 @@ export class AccordionItem implements ConditionalSlotComponent {
   /** Displays the `iconStart` and/or `iconEnd` as flipped when the element direction is right-to-left (`"rtl"`). */
   @Prop({ reflect: true }) iconFlipRtl: FlipContext;
 
+  /**
+   * Specifies the placement of the icon in the header inherited from the `calcite-accordion`.
+   *
+   * @internal
+   */
+  @Prop({ reflect: false }) iconPosition: Position = "end";
```

Comment:
@Elijbet `reflect: false` is the default so I don't think this needs to be specifically defined in the code. Applies to other props.

Reply:
For these new internal props, do they need to have a default set or could they be undefined by default?